starter-code/src/App.js

```diff
@@ -1,18 +1,88 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import contacts from './contacts.json';
+import RowTable from './components/RowTable.jsx'
+
+
+
 
 class App extends Component {
+
+  state = {
+    contacts: contacts.splice(0, 5),
+  };
+
+
+  deleteContact = this.deleteContact.bind(this);
+
```

Comment:
An alternative way to use your deleteContact without using "bind" would be `<RowTable key={index} deleteContact={() => this.deleteContact(index)} />` then in your RowTable.jsx you would need to update your code like this `<button onClick={props.deleteContact}`